src/overlays/actors/ovl_En_Part/z_en_part.h

```diff
@@ -5,10 +5,22 @@
 
 struct EnPart;
 
+typedef void (*EnPartActionFunc)(struct EnPart*, PlayState*);
+
+typedef enum {
+    ENPART_PARAMS_0 = 0,
+    ENPART_PARAMS_1 = 1,
+    ENPART_PARAMS_4 = 4,
+    ENPART_PARAMS_15 = 15
+} ActorParams;
```

Comment:
```suggestion
} EnPartParams;
```
Should also change the enum name.